backend/src/schema/types/type/Post.gql

```diff
@@ -1,26 +1,101 @@
+input _PostFilter {
+  AND: [_PostFilter!]
+  OR: [_PostFilter!]
+  id: ID
+  id_not: ID
+  id_in: [ID!]
+  id_not_in: [ID!]
+  author: _UserFilter
+  author_not: _UserFilter
+  author_in: [_UserFilter!]
+  author_not_in: [_UserFilter!]
+  title: String
+  title_not: String
+  title_in: [String!]
+  title_not_in: [String!]
+  title_contains: String
+  title_not_contains: String
+  title_starts_with: String
+  title_not_starts_with: String
+  title_ends_with: String
+  title_not_ends_with: String
+  slug: String
+  slug_not: String
+  slug_in: [String!]
+  slug_not_in: [String!]
+  slug_contains: String
+  slug_not_contains: String
+  slug_starts_with: String
+  slug_not_starts_with: String
+  slug_ends_with: String
+  slug_not_ends_with: String
+  content: String
+  content_not: String
+  content_in: [String!]
+  content_not_in: [String!]
+  content_contains: String
+  content_not_contains: String
+  content_starts_with: String
+  content_not_starts_with: String
+  content_ends_with: String
+  content_not_ends_with: String
+  image: String
+  visibility: Visibility
+  visibility_not: Visibility
+  visibility_in: [Visibility!]
+  visibility_not_in: [Visibility!]
+  language: String
+  language_not: String
+  language_in: [String!]
+  language_not_in: [String!]
+  pinned: Boolean # required for `maintainPinnedPost`
+  tags: _TagFilter
+  tags_not: _TagFilter
+  tags_in: [_TagFilter!]
+  tags_not_in: [_TagFilter!]
+  tags_some: _TagFilter
+  tags_none: _TagFilter
+  tags_single: _TagFilter
+  tags_every: _TagFilter
+  categories: _CategoryFilter
+  categories_not: _CategoryFilter
+  categories_in: [_CategoryFilter!]
+  categories_not_in: [_CategoryFilter!]
+  categories_some: _CategoryFilter
+  categories_none: _CategoryFilter
+  categories_single: _CategoryFilter
+  categories_every: _CategoryFilter
+  comments: _CommentFilter
+  comments_not: _CommentFilter
+  comments_in: [_CommentFilter!]
+  comments_not_in: [_CommentFilter!]
+  comments_some: _CommentFilter
+  comments_none: _CommentFilter
+  comments_single: _CommentFilter
+  comments_every: _CommentFilter
+  emotions: _PostEMOTEDFilter
+  emotions_not: _PostEMOTEDFilter
+  emotions_in: [_PostEMOTEDFilter!]
+  emotions_not_in: [_PostEMOTEDFilter!]
+  emotions_some: _PostEMOTEDFilter
+  emotions_none: _PostEMOTEDFilter
+  emotions_single: _PostEMOTEDFilter
+  emotions_every: _PostEMOTEDFilter
+}
+
 enum _PostOrdering {
   id_asc
   id_desc
-  activityId_asc
-  activityId_desc
-  objectId_asc
-  objectId_desc
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-18T15:19:01Z" title="Monday, November 18th 2019, 4:19:01 pm +01:00">Nov 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,26 +1,101 @@
+input _PostFilter {
+  AND: [_PostFilter!]
+  OR: [_PostFilter!]
+  id: ID
+  id_not: ID
+  id_in: [ID!]
+  id_not_in: [ID!]
+  author: _UserFilter
+  author_not: _UserFilter
+  author_in: [_UserFilter!]
+  author_not_in: [_UserFilter!]
+  title: String
+  title_not: String
+  title_in: [String!]
+  title_not_in: [String!]
+  title_contains: String
+  title_not_contains: String
+  title_starts_with: String
+  title_not_starts_with: String
+  title_ends_with: String
+  title_not_ends_with: String
+  slug: String
+  slug_not: String
+  slug_in: [String!]
+  slug_not_in: [String!]
+  slug_contains: String
+  slug_not_contains: String
+  slug_starts_with: String
+  slug_not_starts_with: String
+  slug_ends_with: String
+  slug_not_ends_with: String
+  content: String
+  content_not: String
+  content_in: [String!]
+  content_not_in: [String!]
+  content_contains: String
+  content_not_contains: String
+  content_starts_with: String
+  content_not_starts_with: String
+  content_ends_with: String
+  content_not_ends_with: String
+  image: String
+  visibility: Visibility
+  visibility_not: Visibility
+  visibility_in: [Visibility!]
+  visibility_not_in: [Visibility!]
+  language: String
+  language_not: String
+  language_in: [String!]
+  language_not_in: [String!]
+  pinned: Boolean # required for `maintainPinnedPost`
+  tags: _TagFilter
+  tags_not: _TagFilter
+  tags_in: [_TagFilter!]
+  tags_not_in: [_TagFilter!]
+  tags_some: _TagFilter
+  tags_none: _TagFilter
+  tags_single: _TagFilter
+  tags_every: _TagFilter
+  categories: _CategoryFilter
+  categories_not: _CategoryFilter
+  categories_in: [_CategoryFilter!]
+  categories_not_in: [_CategoryFilter!]
+  categories_some: _CategoryFilter
+  categories_none: _CategoryFilter
+  categories_single: _CategoryFilter
+  categories_every: _CategoryFilter
+  comments: _CommentFilter
+  comments_not: _CommentFilter
+  comments_in: [_CommentFilter!]
+  comments_not_in: [_CommentFilter!]
+  comments_some: _CommentFilter
+  comments_none: _CommentFilter
+  comments_single: _CommentFilter
+  comments_every: _CommentFilter
+  emotions: _PostEMOTEDFilter
+  emotions_not: _PostEMOTEDFilter
+  emotions_in: [_PostEMOTEDFilter!]
+  emotions_not_in: [_PostEMOTEDFilter!]
+  emotions_some: _PostEMOTEDFilter
+  emotions_none: _PostEMOTEDFilter
+  emotions_single: _PostEMOTEDFilter
+  emotions_every: _PostEMOTEDFilter
+}
+
 enum _PostOrdering {
   id_asc
   id_desc
-  activityId_asc
-  activityId_desc
-  objectId_asc
-  objectId_desc
```


just temporarily?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-18T20:20:13Z" title="Monday, November 18th 2019, 9:20:13 pm +01:00">Nov 18, 2019</time>_

---


Yes, until we might need it for ActivityPub.